src/ray/object_manager/ownership_based_object_directory.h

```diff
@@ -126,7 +132,11 @@ class OwnershipBasedObjectDirectory : public IObjectDirectory {
   std::function<void(const ObjectID &, const rpc::ErrorType &)> mark_as_failed_;
 
   /// A buffer for batch object location updates.
```

Comment:
Can you expand this comment a bit? It's a bit hard to tell what the data structure is doing and why we care about achieving FIFO.